src/ripple/overlay/impl/TrafficCount.h

```diff
@@ -187,6 +188,7 @@ class TrafficCount
         { "overhead: manifest" },                                 // category::manifests
         { "transactions" },                                       // category::transaction
         { "proposals" },                                          // category::proposal
+        { "validator lists" },                                    // category::validatorlist
```

Comment:
This particular change breaks #3179. It's worth considering whether that needs to be fixed _there_ or here (although considering that it's already included in the proposed **1.5.0-b2** PR #3199, the fix will likely have to come in a separate PR and be coordinated with this).

Reply:
I'm fine with waiting until b2 is pushed, and then resolving conflicts, but I'm looking at #3179, and I don't see how this breaks it. What am I missing?

Reply:
It passes the metrics as one large array and _implicitly_ relies on ordering to know what's what.  This breaks because you've inserted something in the middle of the array.

With that said, it's #3179 that's, ultimately, the problem: it has an extremely fragile implementation and you just happened to the unlucky one to break it. So this needs to be fixed there.

Reply:
It looks like #3179 got pulled from b2, so I guess this isn't as pressing. It would be easy enough to move the new item to the end of the list, but is it necessary, or should I just leave it be and let #3179 get fixed?